config/notifies.go

```diff
@@ -15,62 +15,50 @@ package config
 
 import (
 	"fmt"
+	"strings"
 )
 
 var (
 	// DefaultHipchatConfig defines default values for Hipchat configurations.
 	DefaultHipchatConfig = HipchatConfig{
-		ColorFiring:   "purple",
-		ColorResolved: "green",
+		Color:         `{{ if eq .Status "firing" }}purple{{ else }}green{{ end }}`,
```

Comment:
Actually fine with that. This was just the first one I saw. But it's consistent what is happening in other places below :+1: